examples/plot_sunspot.py

```diff
@@ -0,0 +1,44 @@
+"""
+Importing Sunspot Data
+==============

```

Comment:
The underline needs to be the same length as the title.

Reply:
Got it! Making changes!